tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "module": "commonjs",
+    "module": "esnext",
```

Comment:
Don't need to modify this.

Reply:
It is modified automatically...

Reply:
Mine was modified automatically as well if I do `npm run start` or `npm run` so maybe just don't commit this file.